sr_hand_kinematics/hand_kinematics/include/hand_kinematics/hand_kinematics_plugin.h

```diff
@@ -66,10 +66,10 @@
 
 namespace hand_kinematics
 {
-class HandKinematicsPlugin : public kinematics::KinematicsBase
+  class HandKinematicsPlugin :
```

Comment:
I can't find the option to do that in the clion formatter + roslint is not sending an error :)